roles/advanced-core/advanced_dhcp_server/templates/dhcpd.subnet.conf.j2

```diff
@@ -32,14 +32,15 @@
       {% set ipxe_rom_filename = (ipxe_architecture + '/standard_' + ipxe_driver + 'ipxe.efi') %}
     {% else %}  {# Assume everything else is pcbios #}
       {% set ipxe_rom_filename = (ipxe_architecture + '/standard_undionly.kpxe') %}
-    {% endif %}
+    {% endif -%}
```

Comment:
The minus sign should remove whitespace after the block, but there is no whitespace here?